src/main/java/ch/njol/skript/util/BlockStateBlock.java

```diff
@@ -361,6 +361,11 @@ public void tick() {
 		state.getBlock().tick();
 	}
 
+	@Override
```

Comment:
Don't put the override annotation on the new methods so we can compile it against older server versions without an error.

Reply:
As mentioned on Discord, this will not compile on older versions regardless as the methods for the backing Block do not exist.